doc/source/getting_started/install.rst

```diff
@@ -362,6 +362,19 @@ pyarrow                   0.15.0             Parquet, ORC, and feather reading /
 pyreadstat                                   SPSS files (.sav) reading
 ========================= ================== =============================================================
 
+.. warning::
+
+    * If you want to use :func:`~pandas.read_orc`, it is highly recommended to install pyarrow using conda.
+      The following is a summary of the environment in which :func:`~pandas.read_orc` can work.
+
+      ========================= ================== =============================================================
+      System                    Conda              PyPI
+      ========================= ================== =============================================================
+      Linux                     Successful         Failed(pyarrow==3.0 excepted)
```

Comment:
for Linux on PyPI can you just say pyarrow>=3.0 Successful

Reply:
Yes, I tested it on Linux(ubuntu18.04)/macOS/Windows10.
The above summary is based on the test results.

ps. [The latest version of PyArrow(3.0)](https://pypi.org/project/pyarrow/#history) installed from PyPI works well on Linux. But I'm not sure if the above error will happen in future releases because [JIRA-7811](https://issues.apache.org/jira/browse/ARROW-7811) has not been fixed yet.